tests/ui-toml/min_ident_chars/min_ident_chars.stderr

```diff
@@ -1,8 +1,8 @@
-error: this ident is too short (3 <= 3)
-  --> $DIR/min_ident_chars.rs:6:19
+error: this ident is too short (1 <= 3)
+  --> $DIR/min_ident_chars.rs:6:30
    |
-LL | use extern_types::Aaa;
-   |                   ^^^
+LL | use extern_types::{Aaa, N as W};
```

Comment:
There were 2 distinct changes which that diff does poorly reflect:
* I started by making sure that `use extern_types::Aaa` does not trigger the warning
* I added another identifier with an alias, to make sure the alias triggers the warning